authors/apps/articles/test_articles_ratings.py

```diff
@@ -0,0 +1,351 @@
+from ..base_test import BaseTest
+from django.urls import reverse
+from rest_framework.views import status
+import json
+
+class ArticleTestCase(BaseTest):
+    """ 
+    Class implements tests for artcles
+    """
+    
+    def create_article(self, token, article):
+        """
+        Helper method to creates an article
+        """
+        return self.client.post(
+            '/api/articles',
+            article,
+            HTTP_AUTHORIZATION='Bearer ' + token,
+
+            format='json'
+        )
+
+
+    def rate_article(self, token, slug, rate):
+        """
+        Helper method to creates an article
+        """
+        return self.client.post(
+            '/api/articles/' + slug + '/rate/', rate,
+            HTTP_AUTHORIZATION='Bearer ' + token,
+
+            format='json'
+        )
+
+    
+    def test_cannot_rate_own_article(self):
+        """ 
+        Test an article can be searched by title
+        and returned successfully
+
+        """
+
+        response = self.client.post(
+               self.SIGN_UP_URL,
+               self.user_cred,
+               format='json')
+        response = self.client.post(
+               reverse('authentication:user_login'),
+               self.user_cred,
+               format='json')
+        token = response.data['token']
+
+        article= {
+                    "article": 
+                      {
+                      "title": "How to feed your dragon",
+                      "description": "Wanna know how?",
+                      "body": "You don't believe?",
+                        }
+                  }
+        
+        response = self.create_article(token, article)
+        slug = 'how-to-feed-your-dragon'
+        rate={
+            "rate": {
+                "rating": 5
+            }
+            }
+        response = self.rate_article(token, slug, rate)
+        self.assertEquals(status.HTTP_401_UNAUTHORIZED, response.status_code)
+
+
+    def test_can_rate_article_successfully(self):
+        """ Tests whether a user can delete an article when not author
+        Method registers a user, logs in the user,
+        creates an article, then updates the article
+        """
+
+        response = self.client.post(
+            self.SIGN_UP_URL,
+            self.user_cred,
+            format='json')
+        response = self.client.post(
+            reverse('authentication:user_login'),
+            self.user_cred,
+            format='json')
+        token = response.data['token']
+
+        article = {
```

Comment:
Take this article declaration to the base file so that it can be reused or edited with ease.